exploration/exact-match-selector-options.md

```diff
@@ -0,0 +1,161 @@
+# Design Proposal Template
+
+Status: **Accepted**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2023-12-08</dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+We need to choose a name for the selector function used when no other selector is specified.
+
+## Background
+
+_What context is helpful to understand this proposal?_
+
+This addresses issue <a href="https://github.com/unicode-org/message-format-wg/issues/433">#433</a>
+and issues raised in various design documents, notably
+<a href="https://github.com/unicode-org/message-format-wg/pull/471">#471</a>
+(about number selection).
+
+The exact match selector function is the function used when an implementation
+cannot find another selector.
+It is also an attribute of selectors, such as `:number` or perhaps `:date`
+that need to match specific values.
```

Comment:
```suggestion
The exact match selector function is the simplest selection function,
which compares its input to the variant keys using string equality.

Other selectors, such as `:number` or perhaps `:date`
may provide similar functionality for their supported value types.
```

Reply:
Minor tweaks to this suggestion.

```suggestion
The exact match selector function is the simplest selector function,
which compares its operand to the variant keys using string equality.

Other selectors, such as `:number` and `:date` usually
provide similar functionality for their supported value types.
```

Note that both your (@eemeli) suggestion and mine lose the "defaultness" of this selector. Do you disagree with saying that this is the selector used when no other selector's signature matches?

Reply:
Yeah. It looks like we accidentally dropped a line in #441 that was added in #431: https://github.com/unicode-org/message-format-wg/blob/812c6dde8e8834341aae42198256f76e399f9a8c/spec/syntax.md?plain=1#L313-L314

In other words, we've resolved not to have a fallback or default selector. Instead, one must always be defined. I'll add a PR re-introducing that line.

Reply:
Otherwise, your suggestion is good, and with that I'm happy to approve this.